packages/fxa-admin-server/src/gql/account/account.resolver.ts

```diff
@@ -142,6 +142,21 @@ export class AccountResolver {
     return !!result;
   }
 
+  @Mutation((returns) => Boolean)
+  public async recordAdminSecurityEvent(
+    @Args('uid') uid: string,
+    @Args('name') name: string
+  ) {
+    // the ipAddr and ipHmacKey values here are required, but also have no bearing on this type of record.
+    // the securityEvents table is being repurposed to store a broader variety of events, hence the dummy values.
+    await this.db.securityEvents.create({
+      uid,
+      name: 'emails.clearBounces',
+      ipAddr: '0',
```

Comment:
This seems alright to do, since we don't have the user's IP and we are acting on behalf of the user. Another possibility is to record the IP of the admin panel user making the current request. I can see arguments  both ways, but setting to 0 does seem the simplest. 

@clouserw do you have any opinions on this?

Reply:
I think it should be null/blank.

Reply:
If we set it to null or blank, will that cause non-unique values being used for the `INDEX` on the `securityEvents` table? I think the `ipAddr` is used as a key to make the `ipAddrHmac` and then the index is made from a combination of `uid` and `ipAddrHmac`, so I worry that any static value would cause a collision (but maybe I'm missing something about the `ipAddrHmac` is made?)

Reply:
if a UID and an IP are being used as an index you're already going to be running into collisions.  I assume the hmac will have some protection against it, but it's a good idea to investigate it.  Good thinking!

Reply:
I wasn't able to determine HOW we safely use the created value in an index, but I did replicate the WAY that we do it in the auth server (passing in the key from a config.) Does that feel like due diligence?

Reply:
Just a side note, this is safe from collisions. Although `ipAddrHmac` is part of an index,(`KEY 'securityEvents_uid_ipAddrHmac_createdAt' (uid,ipAddrHmac,createdAt)`) you'll notice there is no unique or primary key constraints on this table, so a collision can't really happen. 

_(It's a bit odd we don't have a primary key here, and this is generally not recommended, but perhaps there was a reason.)_